src/radio-button.jsx

```diff
@@ -57,7 +57,7 @@ const RadioButton = React.createClass({
 
   propTypes: {
     /**
-     * Used internally by `RadioButtonGroup`.
+     * @ignore
      */
     /* Checked if true. */
```

Comment:
okay @oliviertassinari I'm removing the below comment and adding it in the first one